docs/user-guide/global-configurations/chart-repo.md

```diff
@@ -0,0 +1,60 @@
+# Chart Repository
+
+This feature allows you to add more chart repositories to Devtron. Once added they will be available in the Discover section of the Chart Store. 
+
+Learn more about [Discover chart link](https://docs.devtron.ai/user-guide/deploy-chart/overview-of-charts#deploying-chart)
+
+## Add Chart Repository
+
+Select the Chart Repository section of global configuration and click on `Add Repository` button at the top of the Chart Repository Section to add new chart, you need to provide three inputs as below.
+
+1. Name
+2. URL
+3. Authentication type
+
+![](../../.gitbook/assets/gc-add-chart.png)
+
+### 1. Name
+
+Provide a `Name` to your Chart Repository. This name is added as prefix to the name of the chart in the listing on the helm chart section of application.
+
+![](../../.gitbook/assets/gc-chart-name-highlight.png)
+
+### 2. URL
+
+Provide the `URL`. **For example**- github.com for Github, [https://gitlab.com](https://gitlab.com) for GitLab, etc.
+
+### 3. Authentication type
+
+Here you have to provide the type of Authentication required by your version controller. We are supporting three types of authentications, you can choose any one from them.
+
+
+* **Anonymous**
+
+If you select `Anonymous` then you do not have to provide any username, password, and authentication token. Just click on `Save` to save your chart repository details.
```

Comment:
If you select `Anonymous` then you do not have to provide any username, password, or authentication token. Just click on `Save` to save your chart repository details.